frontend/src/features/admin/edit-user/EditUserForm.tsx

```diff
@@ -46,7 +46,7 @@ const EditUserForm: React.FunctionComponent<IEditUserFormProps> = ({
     >
       {formikProps => (
         <Form className="userInfo">
-          <SectionField label="IDIR/BCeID" wideScreen>
+          <SectionField label="IDIR/BCeID" labelWidth="2">
```

Comment:
changes not directly related to the documents screen. Changed for consistency and flexibility of the SectionField